.github/workflows/chipdb.yml

```diff
@@ -9,67 +9,82 @@ jobs:
+        docker pull pepijndevos/apicula:1.9.8
+        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:1.9.8 make apycula/GW1N-1.pickle
     - name: Archive artifact
       uses: actions/upload-artifact@v2
       with:
           name: gw1n-1-chipdb
           path: apycula/GW1N-1.pickle
-  gw1ns2:
+    - name: Archive stage artifact
+      uses: actions/upload-artifact@v2
+      with:
+          name: gw1n-1-stage
+          path: GW1N-1*
+  gw1n9:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build gw1ns-2 chipdb
+    - name: Build gw1n-9 chipdb
       run: |
-        docker pull pepijndevos/apicula:latest
-        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:latest make apycula/GW1NS-2.pickle
+        docker pull pepijndevos/apicula:1.9.8
+        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:1.9.8 make apycula/GW1N-9.pickle
     - name: Archive artifact
       uses: actions/upload-artifact@v2
       with:
-          name: gw1ns-2-chipdb
-          path: apycula/GW1NS-2.pickle
+          name: gw1n-9-chipdb
+          path: apycula/GW1N-9.pickle
+    - name: Archive stage artifact
+      uses: actions/upload-artifact@v2
+      with:
+          name: gw1n-9-stage
+          path: GW1N-9*
   gw1n4:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Build gw1n-4 chipdb
       run: |
-        docker pull pepijndevos/apicula:latest
-        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:latest make apycula/GW1N-4.pickle
+        docker pull pepijndevos/apicula:1.9.8
+        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:1.9.8 make apycula/GW1N-4.pickle
     - name: Archive artifact
       uses: actions/upload-artifact@v2
       with:
           name: gw1n-4-chipdb
           path: apycula/GW1N-4.pickle
-  gw1n9:
+    - name: Archive stage artifact
+      uses: actions/upload-artifact@v2
+      with:
+          name: gw1n-4-stage
+          path: GW1N-4*
+  gw1ns2:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build gw1n-9 chipdb
+    - name: Build gw1ns-2 chipdb
       run: |
-        docker pull pepijndevos/apicula:latest
-        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:latest make apycula/GW1N-9.pickle
+        docker pull pepijndevos/apicula:1.9.8
+        docker run -v $(pwd):/usr/src/apicula pepijndevos/apicula:1.9.8 make apycula/GW1NS-2.pickle
     - name: Archive artifact
       uses: actions/upload-artifact@v2
       with:
-          name: gw1n-9-chipdb
-          path: apycula/GW1N-9.pickle
+          name: gw1ns-2-chipdb
+          path: apycula/GW1NS-2.pickle
+    - name: Archive stage artifact
+      uses: actions/upload-artifact@v2
+      with:
+          name: gw1ns-2-stage
+          path: GW1NS-2*
   pypi:
     runs-on: ubuntu-latest
-    needs: [gw1n9, gw1n1, gw1n4, gw1ns2]
+    needs: [gw1n1, gw1n9, gw1n4, gw1ns2]
```

Comment:
what's all this reshuffling about??

Reply:
my bad: GW1N-1 is always being experimented on, so sometimes it's just one in the list.